test/integration/location.js

```diff
@@ -22,12 +22,12 @@ test('output to the correct location', async (t) => {
       filename: '[name].js',
       path: outputPath
     },
-    plugins: [new WebpackManifestPlugin({ fileName: 'webpack.manifest.js' })]
+    plugins: [new WebpackManifestPlugin({ fileName: 'webpack.manifest.json' })]
```

Comment:
If you end the file in `.js`, the TerserPlugin tries to parse these files and explodes. I'm not sure why this is different between v4 and v5 (it's only a problem on v5), but it makes sense: the manifest contents are not actually a valid JavaScript file (`{ "foo": "bar" }` is not valid JavaScript - you'd need to have a `const = ` in front or something equal).

And so, I think this is valid to change. Users are able to control the filename (the point of the test), but in Webpack 5, they cannot name it `.js`.

Reply:
Is that something we should document in the README?

Reply:
Hmm. I would say, let's wait to see if we have any questions about it. The only reason I'm hesitating is that the JSON was *never* a valid JavaScript file... so I'm trying to think of what the use-case ever would have been (and so, is anyone out there actually doing that?).